vocab_files/attribute_concepts/height-above-trap-cover-.ttl

```diff
@@ -9,6 +10,7 @@ PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
     rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
     skos:definition "The height (in cms) recorded above the trap the cover is installed." ;
     skos:prefLabel "height above trap cover " ;
+    schema:url "https://github.com/ternaustralia/dawe-rlp-vocabs/tree/master/vocab_files/attribute_concepts/height-above-trap-cover-.ttl" ;
```

Comment:
hi Edmond, there is an extra `-` after `height-above-trap-cover`

Reply:
Good catch. Looks like the label has a whitespace at the end of it.